src/vue/backend/src/styles/verticalNavBar.scss

```diff
@@ -1,19 +1,4 @@
 // The incorrect usage of BEM lines up with the VCL implementation. Will need a major VCL upgrade to change.
-$v-nav-width: 255px;
```

Comment:
These variables are specific to the navigation implementation, and are unlikely to be used elsewhere. Therefore, they should remain in the component and not moved to the global variable

Reply:
I moved it here because I used it in another place